docs/ajaxApi.rst

```diff
@@ -262,7 +262,7 @@ Here's a curl command sample:
 
 .. code-block:: guess
 
-   curl -k -i -H "Content-Type: multipart/mixed" -X POST --form 'session.id=e7a29776-5783-49d7-afa0-b0e688096b5e' --form 'ajax=upload' --form 'file=@myproject.zip;type=application/zip' --form 'project=MyProject;type/plain' https://localhost:8443/manager
```

Comment:
Works for me:
<img width="1467" alt="Screen Shot 2019-10-28 at 2 11 02 PM" src="https://user-images.githubusercontent.com/44478711/67718405-f0129c80-f98c-11e9-96d4-544cc9114311.png">

But we do need to replace ";type/plain" with ";type=text/plain"

Reply:
![image](https://user-images.githubusercontent.com/10431666/67934494-f742e000-fc02-11e9-870b-ab8d2f3c081f.png)

I replaced ";type/plain" with ";type=text/plain", but still error, any suggestion?

Reply:
I'm not able to reproduce this authentication error but let's make the change anyway as multipart/form-data is better suited here and more commonly supported.